data/darktableconfig.xml.in

```diff
@@ -183,13 +183,11 @@
         <option>default</option>
         <option>large</option>
         <option>unrestricted</option>
-        <option>mini (debug)</option>
-        <option>reference (debug)</option>
       </enum>
     </type>
     <default>default</default>
     <shortdescription>darktable resources</shortdescription>
-    <longdescription>defines how much darktable may take from your system resources.\n - default: darktable takes ~50%% of your systems resources and gives darktable enough to be still performant.\n - Use `small` if you are simultaneously using applications taking large parts of your systems memory or opencl/gl applications like games or hugin.\n - Use `large` if you want darktable to take most of your systems resources for performance. Probably best option while exporting images.\n\nspecial modes used for tests and debugging:\n\n - mini: darktable takes 0.5 GB ram and 0.25GB video ram\n - reference: - darktable takes 8GB ram and 2GB video ram\n - unrestricted: darktable will take almost all of your systems resources and thus might lead to swapping and unexpected performance drops. use with caution!</longdescription>
+    <longdescription>defines how much darktable may take from your system resources.\n - default: darktable takes ~50%% of your systems resources and gives darktable enough to be still performant.\n - Use `small` if you are simultaneously using applications taking large parts of your systems memory or opencl/gl applications like games or hugin.\n - Use `large` if you want darktable to take most of your systems resources for performance. Probably best option while exporting images.\n - unrestricted: darktable will take all of your systems resources and thus might lead to swapping and unexpected performance drops. use with caution!</longdescription>
```

Comment:
Can you make these consistent please -- either start everything in your list with "option: description" or "Use `option` if you..."

Reply:
What would you prefer style-wise - as in a manual?

Reply:
I think "option: description" is clearer, though I understand it's often harder to write nice-reading stuff like that. I won't be much more picky than that -- I can always push a copy-edit PR later if needed.

Reply:
i guess the last commit makes this better.

Reply:
you marked as "documentation pending".

I think the whole resourcelevel thing is now stable, i will write something about 1) why this has been changed in darktable 2) for what scenarios the levels are meant 3) how the tuning-cl works and 4) how you could do a manual correction of the fractions. Anything left out here? More or less technical, what do you think is good for the manual?

Reply:
Yeah. When we get round to writing dtdocs for this I'll make sure to unify it with the tips as much as possible. No point paying too much attention to the wording until this has had some decent field-testing though.